packages/blade/src/components/Collapsible/CollapsibleBody.tsx

```diff
@@ -0,0 +1,12 @@
+import type { ReactElement, ReactNode } from 'react';
+import { CollapsiblePanel } from './CollapsiblePanel';
+
+type CollapsibleBodyProps = {
+  children: ReactNode;
+};
+
+const CollapsibleBody = ({ children }: CollapsibleBodyProps): ReactElement => {
```

Comment:
Planning to move common things from `CollapsiblePanel.web` and `CollapsiblePanel.native` here, also used this later for adding meta attributes and box wrapper

Reply:
Refactored CollapsiblePanel -> CollapsibleBodyContent in #1296. Because I also want to have CollapsibleBody separately for passing common things / wrapper. I think should help in readability of which component goes inside which.